tests/cli/test_suite.py

```diff
@@ -3135,7 +3135,11 @@ def test_suite_new_profile_runs_notebook_no_jupyter(
     # },
     # column_value_nonnullity_rule={
     # },
-    # numeric_columns_rule={
+    # numeric_columns_low_precision_rule={
+    #     "false_positive_rate": 0.1,
+    #     "random_seed": 43792,
+    # },
```

Comment:
@NathanFarmer How come `round_decimals` is absent here (it is rightfully present in the rendered file.  Thanks!